news/2020-12-09-new-featured-artist-lifetheory.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Lifetheory"
+date: 2020-12-09 09:00:00 +0000
+---
+
+**Lifetheory** joins our Featured Artist lineup as our latest addition, and that's a fact!
+
+![](https://assets.ppy.sh/artists/122/header.jpg)
+
+Fusing a whole slew of genres into something new and wonderful, **Lifetheory** is an insanely talented Russian producer sporting a similarly wide range of musical influences. Drawing elements from the likes of *Pendulum*, *Hanz Zimmer* and *Linkin Park*, his music effortlessly strides into aural territory rarely tread.
+
+With **19** new tracks now available, there's no shortage of epic electronic experiences waiting in the wings, so why not give some a try?
+
+In case you weren't with us yet when the **Lifetheory** craze was at its peak, check it all out in a little medley style preview below, courtesy of the most pishi of fats:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/122/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+Still need more convincing? We've got you covered - check out the trilogy of maps that started it all:
+
+### Daisy
+
+Check out [this classic 2013 set](https://osu.ppy.sh/beatmapsets/118494) mapped by [Zare](https://osu.ppy.sh/users/1437117)!
```

Comment:
I think it'd read nicer to include the "mapped by zare" and "classic set" parts in L23, and leave these sections empty besides the audio. cuts out the repetition of "classic set" and "trilogy"

if applied I guess that means the headers would have to be the map links instead